#include "Service/RegisterAllocationInference/RegisterAllocationInference.grpc.pb.h"
#include <grpcpp/grpcpp.h>
#include <vector>


namespace Inference {

using grpc::Status;
using grpc::ServerContext;
using registerallocationinference::GraphList;
using registerallocationinference::ColorData;
using registerallocationinference::RegisterAllocationInference;

//void sendGraphs(int argc,char**argv,RegisterAllocationInference::Stub*);
//void genCode(int argc,char**argv,std::string predData);

}

